CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+- Add `impl_display_{states,events}` for auto-impementation of `core::fmt::Display` on `States` and/or `Events`
```

Comment:
Can you mention that this can be enabled by specifying them as a keyword in the sm! macro? The docs for smlang-rs are lacking right now, so this would be the best place to tell users about this feature.

Reply:
Yeah, that's a very valid point. I'd love if we had better docs on features. Please feel free to add it wherever you think it makes the most sense!